src/coffee/mapping-utils/csv.coffee

```diff
@@ -76,18 +96,37 @@ class CsvMapping
       return code
     ).join(';')
   # TODO: Move method below to sphere-node-utils
+
+  formatProductAttributes: (attributeName, attributes) ->
+    mappedAttributes = @productMapper._mapAttributes(attributes)
+    value = mappedAttributes[attributeName]
+
+    if value is undefined
```

Comment:
question: why are we checking against undefined instead of `!value` ?

Reply:
Hello @daern91 .. this is because the `value` can contain a zero which would be mapped to an empty string.

Reply:
I added a comment here 867eadc